helm/hpcc/values.schema.json

```diff
@@ -447,10 +440,10 @@
           "type": "array",
           "items": { "type": "string" }
         },
-        "labels": {
-          "description": "a list of labels associated with this plane, e.g. lz, data, dali, sasha, dlls, spill. NB: if blank/unset, treated as if 'data'",
-          "type": "array",
-          "items": { "type": "string" }
+        "category": {
+          "description": "the category this plane is usd for, e.g. lz, data",
+          "type": "string",
+          "enum": ["data", "lz", "dali", "sasha", "dll", "spill" ]
```

Comment:
what was the conclusion on new categories?
is spill all temporary data, including inter sub-graph, internal activity spilling (e.g. hash dedup buckets)?
and we also talked about published job owned temporary results.

Possibly for another JIRA/PR anyway

Reply:
I think we add new categories when we have a use for them.  spill isn't actually used at the moment - we need to go through and implement that.  If we want multiple categories for spills we can add them then.